README.md

```diff
@@ -1,7 +1,6 @@
 <div align="center">
 
-  <img src="assets/squadjs-logo-white.png#gh-dark-mode-only" alt="Logo" width="500"/>
-<img src="assets/squadjs-logo.png#gh-light-mode-only" alt="Logo" width="500"/>
```

Comment:
It looks like this change was a mistake?

Reply:
I didn't change that intentionally. 

I also thought I pulled from squadjs latests.. I wonder if something in the linter/yarn scripts is acting up on my end, I've noticed whitespace changes after each commit for a while.

Reply:
Possibly, can you try reverting it please?

Reply:
I think this is husky rebuilding the readme file, checking to confirm

Reply:
Try merging in master and trying again - it looks like I forgot to add the light logo to the README template.